src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,103 @@
+package duke.command;
+
+import duke.exceptions.DukeException;
+import duke.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    public static final String HORIZONTAL_LINE_TOP = "\n______________________________" +
+            "______________________________\n";
+    public static final String HORIZONTAL_LINE_BOTTOM = "_______________________________" +
+            "_____________________________\n";
+    private static boolean hasUserExited = false;
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static String file = "data/duke.txt";
+
+    public static void checkFilePath(String filePath) {
```

Comment:
The name is `checkFilePath` however, you are creating the folder if it is not present. So it is a misleading name. Please use a better name for the method.